test/lib/elements/Element.js

```diff
@@ -9,6 +10,28 @@ describe('Element', () => {
         expect(secondVar.previousToken.previousToken.sourceCode).to.equal(';');
     });
 
+    describe('Element#replaceChild', () => {
+        it('should replace child', () => {
+            let program = parseAndGetProgram('var first = 1; var second = 2;');
+            let firstVar = program.selectNodesByType('VariableDeclaration')[0];
+
+            program.replaceChild(new Token('Whitespace', ';'), firstVar);
+
+            expect(program.sourceCode).to.equal('; var second = 2;');
+        });
+
+        it.skip('should replace child with existing one', () => {
```

Comment:
Nope, adding nodes with a parent is forbidden due to huge performance degradation.